src/Form.js

```diff
@@ -0,0 +1,254 @@
+                <label htmlFor="">Avatar:</label>
+              </td>
+              <td>
+                <input
+                  type="file"
+                  value = {avatar}
+                  onChange={this.handleChangeAvatar} />
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <label htmlFor="">Số điện thoại:</label>
+              </td>
+              <td>
+                <input
+                  type="number"
+                  value = {phone}
+                  onChange={this.handleChangePhone} />
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <label htmlFor="">Tuổi:</label>
+              </td>
+              <td>
+                <select
+                  value = {age}
+                  onChange={this.handleChangeAge} >
+                  <option value="1">1 tuổi</option>
+                  <option value="2">2 tuổi</option>
+                  <option value="3">3 tuổi</option>
+                  <option value="4">4 tuổi</option>
+                </select >
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <label htmlFor="">Giới tính:</label>
+              </td>
+              <td>
+                <input
+                  type="radio"
+                  value="male"
+                  checked={gender === "male"}
+                  onChange={this.handleChangeGender}/>Nam<br/>
+                <input
+                  type="radio"
+                  value="female"
+                  checked={gender === "female"}
+                  onChange={this.handleChangeGender}/> Nữ
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <label htmlFor="">Sở thích:</label>
+              </td>
+              <td>
+                <input
+                  type="checkbox"
+                  value="soccer"
+                  defaultChecked={favorite.soccer}
+                  onClick={this.handleChangeSoccer}/> Đá bóng <br/>
+                <input
+                  type="checkbox"
+                  value="badminton"
+                  defaultChecked={favorite.badminton}
+                  onClick={this.handleChangeBadminton}/> Cầu lông <br/>
+                <input
+                  type="checkbox"
+                  value="sailing"
+                  defaultChecked={favorite.sailing}
+                  onClick={this.handleChangeSailing}/> Đua thuyền
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <label htmlFor="">Ghi chú:</label>
+              </td>
+              <td>
+                <textarea
+                  rows="3" cols="25"
+                  value={note}
+                  onChange={this.handleChangeNote}/>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <button
+          type="submit"
+          onClick={this.handleSubmit}>Đăng kí</button>
+      </div>
```

Comment:
Hãy thử focus chuột vào 1 ô input vd focus chuột vào `input name="name"`
Sau đó gõ enter
Xem form có submit không?

Sau khi thử xong, thì gói toàn bộ input trong component này vào 1 thẻ form,
Thẻ form có sự kiện `onSubmit`
Sau đó rút ra kết luận